.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,6 +22,6 @@ Thanks again!
 
 ### Check off the following:
 - [ ] All of the tests are passing.
-- [ ] I have reviewed my changes in staging.
+- [ ] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in the resulting Pull Request).
```

Comment:
```suggestion
- [ ] I have reviewed my changes in staging. Look for the **deploy-to-heroku** link in the resulting pull request.
```

Reply:
```suggestion
- [ ] I have reviewed my changes in staging. Look for the **deploy-to-heroku** link in your pull request, then click **View deployment**.
```
💝 @potatoqualitee I had this update on my to do list and was just not getting to it. Thank you so much for opening a PR! I used @heiskr's suggestion and expanded it to let folks know what button to push. Do you think it's clear?

Reply:
Yes! looks great, @janiceilene ! 🎉 My only concern with the wording is that I goes from first person to third person ("I have" to "Look for"). The parens helped the reader see that it was a hint for them. But it's not a show-stopper.

Reply:
Perhaps `I have looked for the  **deploy-to-heroku** link in my pull request, then clicked **View deployment** and reviewed my changes in staging` though that feels clunky.

Reply:
Ooh, good call 🙌  I just updated my suggestion to include the parenthesis. If that looks good to you, feel free to commit the suggestion and we'll get this merged in soon!

Reply:
This one slipped through the cracks! Sorry about that @potatoqualitee 💛 How do you feel about the language in my suggestion? If you want to commit that suggestion (or change it in some way), I'll get this merged in once you're ready!